airbrakes/mock/mock_imu.py

```diff
@@ -72,25 +73,30 @@ def _fetch_data_loop(self, log_file_path: Path, real_time_simulation: bool) -> N
                 est_linear_accel_x = row.get("estLinearAccelX")  # estimated data packet field
                 # Create the data packet based on the row
                 if scaled_accel_x:
+                    # If the row has the scaledAccelX field, then it's a raw data packet
```

Comment:
already commented that above with the variable

Reply:
woops i think that happened when i did the merge, i'll fix